<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>akademisyen</class>
 <widget class="QWidget" name="akademisyen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="akademisyen_sifre">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>179</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:2px;
border-style:solid;
border-color: rgb(230, 115, 0);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="akademisyen_giris">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color: rgb(230, 115, 0);</string>
   </property>
   <property name="text">
    <string>Giriş Yap</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ŞİFRE:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="akademisyen_adi">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>179</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:2px;
border-style:solid;
border-color: rgb(230, 115, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kullanıcı Adı:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>-10</y>
     <width>201</width>
     <height>89</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#E67300;</string>
   </property>
   <property name="text">
    <string>AKADEMİSYEN GİRİŞİ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="arka_plan">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="giris">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>133</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>arka_plan</zorder>
  <zorder>akademisyen_sifre</zorder>
  <zorder>akademisyen_giris</zorder>
  <zorder>label_2</zorder>
  <zorder>akademisyen_adi</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>giris</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
